### ISSUE TITLE
Do Jobs get killed when we close the app?

### ISSUE THREAD

**Issue by DanPrado** [yigit/android-priority-jobqueue#248]
Hi,
I would like to know if Path jobs are killed when we close the app or when we finish the application process.
If yes, is there a way to replicate Services' `startForeground()` method behaviour?

Thanks!

**Comment by ZakTaccardi** [comment]
I'm sure they're killed, but if you set them to persist they'll run again when the app restarts

**Comment by DanPrado** [comment]
Cool. If we just change to another app (or turn off the screen) and let our app running in background the Job will keep running though, right? Thanks @ZakTaccardi.

**Comment by yigit** [comment]
yes, as long as app process is running, jobs will be run (or ready to be run if constraints match). You can also integrate with schedulers to automatically start the process.

https://github.com/yigit/android-priority-jobqueue/wiki/Integration-with-JobScheduler-and-GcmNetworkManager

**Comment by DanPrado** [comment]
Cool @yigit! Thank you very much.

**Comment by mseshachalam** [comment]
What happens to the jobs that are running when app is closed ?

**Comment by ZakTaccardi** [comment]
They keep running unless the process is killed. If it's killed and you
properly set a retry strategy and the job is persisted to disk, they'll run
again.

**Comment by IvanBean** [comment]
Hi,
I have set job persisted and integrate with schedulers, when app force killed (swipe from recent app), job will never run until app restart, is this normal behavior?

Thanks.

**Comment by kalpeshp0310** [comment]
@IvanBean I think yes. Since job is part of your app and when process gets killed the program execution stops. Job will never run until new app process is created (Either through click on launcher icon or through any other program like broadcast or JobScheduler) and JobManager gets instantiated.

**Comment by mescartin** [comment]
@yigit Do Jobs interact with doze mode like jobScheduler??  If not --->How would jobs interact if you´ve set retry strategies? Will jobs that want to execute at an exact time do something?
